webapp/src/app/core/modules/openapi/api/activity.serviceInterface.ts

```
@@ -46,4 +47,20 @@ export interface ActivityServiceInterface {
      */
     getActivityByUser(login: string, extraHttpRequestParams?: any): Observable<Activity>;
 
+    /**
+     * 
+     * 
+     * @param badPracticeId 
+     * @param badPracticeFeedback 
+     */
+    provideFeedbackForBadPractice(badPracticeId: number, badPracticeFeedback: BadPracticeFeedback, extraHttpRequestParams?: any): Observable<{}>;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)